tests/translator/input/state_machine_with_schedule_role.yaml

```diff
@@ -0,0 +1,15 @@
+Resources:
+
+  MyStateMachine:
+    Type: AWS::Serverless::StateMachine
+
+    Properties:
+      DefinitionUri: s3://sam-demo-bucket/my_state_machine.asl.json
+      Events:
+        CWSchedule:
+          Type: Schedule
+          Properties:
+            Schedule: rate(1 minute)
+            Description: test schedule
+            Enabled: false
+            RoleArn: FakeArn
```

Comment:
should look like an arn at least. `!Sub 'arn:${AWS::Partition}:iam::role/yoyo'` will do